pyvista/utilities/helpers.py

```diff
@@ -1458,3 +1459,56 @@ def cubemap(path='', prefix='', ext='.jpg'):
         texture.SetInputDataObject(i, flip.GetOutput())
 
     return texture
+
+
+def set_default_active_vectors(mesh: 'pyvista.DataSet') -> None:
+    """Set a default vectors on mesh, if not already set.
+
+    If an active vector already exists, no changes are made.
+
+    If an active vectors does not exist, it checks for possibly cell
+    or point arrays with shape ``(n, 3)``.  If only one exists, then
+    it is set as the active vectors.  If none or more than one exists,
+    then a ValueError is raised.
```

Comment:
Aside from the markup, this still says "ValueError". Which is correct, because both errors are ValueErrors. Just pointing this out in case this phrasing was left over from the earlier implementation and you want to change it.
```suggestion
    then a ``ValueError`` is raised.
```

Reply:
We don't need to specify the error at all here, since it is laid out in detail later.